### ISSUE TITLE
Trouble with start of programm

### ISSUE THREAD

**Issue by 2Lazy4Playing** [dustinblackman/Championify#288]
Hi, i have problem with Championify. When new patch camed out, i started the program. After start the background pops up and that's it :P . So... i thought i will re-install championify. I delete folder with program. After that i started download new version of program. I started setup.exe and after that the error pop up (screen is included). Later i started to downlad rar version of it. When i unpack that this same bug pop up (i mean this with background). Can someone help me :)
![przechwytywanie](https://cloud.githubusercontent.com/assets/26468196/24013950/5f1d24fc-0a83-11e7-87ec-19e28fd793af.PNG)

**Comment by dustinblackman** [comment]
What's the setup log say?

**Comment by 2Lazy4Playing** [comment]
Here u have, but it's a lot of text xD
[SquirrelSetup.txt](https://github.com/dustinblackman/Championify/files/849032/SquirrelSetup.txt)

**Comment by dustinblackman** [comment]
That looks like a pretty weird bug with Squirrels, first time I've seen it.

Try deleting the following folders and running the installers again. If it fixes it then I'll pass this on to the Squirrels team.

C:\Users\Kamil\AppData\Local\SquirrelTemp
C:\Users\Kamil\AppData\Local\Championify
C:\Users\Kamil\AppData\Roaming\Championify

**Comment by 2Lazy4Playing** [comment]
i delete those files and empty the thrash can. I started the setup and this same error pops up. I tryed rar version to (same bug with background). But i think i found an solution. Couse in those logs are trying to get to my F:\ Partition. About week ago my disc (with partition F and E) broke, and i'm thinking this can be a problem. Btw i'm giving u logs with this attemp after deleting those files.
[SquirrelSetup2.txt](https://github.com/dustinblackman/Championify/files/849120/SquirrelSetup2.txt)
